docs/spec/6-payloads.md

```diff
@@ -360,6 +361,25 @@ message PairInstallation {
 `MembershipUpdateEvent` is a message used to propagate information about group membership changes in a group chat.
 The details are in the [Group chats specs](./../draft/7-group-chat.md).
 
+### CommunityMessageArchiveIndex
+
+The node uses `CommunityMessageArchiveIndex` messages to distribute message history archive indices to other nodes.
+
+```protobuf
+message CommunityMessageArchiveIndex {
+  uint64 clock = 1;
+  string magnet_uri = 2;
+}
```

Comment:
If we decide to not distribute the index via magnet link, but as-is, then this payload needs to change.